.github/workflows/python-integrate.yaml

```diff
@@ -0,0 +1,172 @@
+# PR's only
+
+name: "Python: Integrate"
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - node/**
+      - go/**
+      - proto/**
+
+env:
+  FORCE_COLOR: 1
+  FORMAT_INSPECT_DEPTH: 20
```

Comment:
This are env vars that improve output from Node.js test runners, Are they needed in Python case?

Reply:
Nope, I had assumed they were set for another purpose in the CI stack. I'll remove them